### PR TITLE
Upgrade to log4j 2.16.0 to address CVE-2021-45046 [nocheck]

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

File: h2o-assemblies/main/build.gradle [hunk 1]
```diff
@@ -83,6 +83,8 @@ shadowJar {
   exclude 'test.properties'
   exclude 'cockpitlite.properties'
   exclude 'devpay_products.properties'
+  exclude 'org/apache/logging/log4j/core/lookup/JndiLookup.class'
+  exclude 'org/apache/logging/log4j/core/lookup/JmxRuntimeInputArgumentsLookup.class'
```
Comment (mmalohlava) [inline h2o-assemblies/main/build.gradle:87]: do you need to remove 2nd class?

**Review by mmalohlava — APPROVED** [review]
ACK, THX!

**Comment by mmalohlava** [comment]
@achraf-mer for info

File: h2o-assemblies/main/build.gradle (hunk 1 above)
Comment (mmalohlava) [reply h2o-assemblies/main/build.gradle:87]: after discussion with @coolsat there is a concern that the change could impact "innocent" customers (who needs JNDI)

Comment (michalkurka) [reply h2o-assemblies/main/build.gradle]: ok, reverting back to just version upgrade

**Pull request merged**